### PR TITLE
Adds the Vice Officer

### PR DESCRIPTION
**Pull request by ghost**
:cl:
add: Added a new job, the Vice Officer. They're basically a more lax Security Officer with cooler clothes.
/:cl:

### CODE REVIEW

File: hippiestation/code/modules/jobs/job_types/vice_officer.dm [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+	total_positions = 2
+	spawn_positions = 2
+	supervisors = "the head of security"
+	selection_color = "#ffeeee"
+	minimal_player_age = 7
+
+	outfit = /datum/outfit/job/vice_officer
+
+	access = list(ACCESS_SECURITY, ACCESS_BRIG, ACCESS_MAINT_TUNNELS, ACCESS_WEAPONS, ACCESS_KITCHEN, ACCESS_BAR, ACCESS_HYDROPONICS)
+	minimal_access = list(ACCESS_SECURITY, ACCESS_BRIG, ACCESS_MAINT_TUNNELS, ACCESS_WEAPONS, ACCESS_KITCHEN, ACCESS_BAR, ACCESS_HYDROPONICS, ACCESS_FORENSICS_LOCKERS, ACCESS_MECH_SECURITY, ACCESS_MINERAL_STOREROOM)
```
Comment (yoyobatty) [inline hippiestation/code/modules/jobs/job_types/vice_officer.dm]: minimal has more access than access

**Comment by yoyobatty** [comment]
Can you explain this a bit more? Maybe see where it fits in with current sec and the meta? Give him a backstory or something.

File: hippiestation/code/modules/jobs/job_types/vice_officer.dm (hunk 1 above)
Comment (ghost) [reply hippiestation/code/modules/jobs/job_types/vice_officer.dm]: Whoops

**Comment by ErwinSnowden** [comment]
Basically originates from goon and pop culture, nigga smokes weed and lets others get high with him if they in the club. But in all honesty, its a sec officer who allows crimes that aren't major.

**Comment by WarRage333** [comment]
> Basically originates from goon and pop culture, nigga smokes weed and lets others get high with him if they in the club. But in all honesty, its a sec officer who allows crimes that aren't major.

Seems fun and good for hippie.
+1

**Comment by karmaisblackandbluepilled** [comment]
why did you put both of these things in the same pr
-1

**Comment by SireTurret** [comment]
+1

**Comment by Manly-Dwarf** [comment]
Why is there a bugfix in a job PR? Also no.

**Comment by Manly-Dwarf** [comment]
Replace Lawyer with Vice honestly

**Comment by ghost** [comment]
@Manly-Dwarf 
> Why is there a bugfix in a job PR? Also no.

I was editing gamemode restricted jobs anyways, might as well 2 birds 1 stone

**Comment by TrickyDickRichard** [comment]
+1 epic

**Comment by ghost** [comment]
Is anyone even going to play as this? Do they get anything sec officers don't have access to?

**Comment by mkikt4743** [comment]
Quarian makes a good point, what do they really do? Would be cool if they were roundstart undercover cops

**Comment by DaydreamPuppeh** [comment]
Split these into 2 PRs

**Comment by ghost** [comment]
fine i've removed the fix

**Comment by cris5311** [comment]
+1
 sure, seems cool

**Comment by karmaisblackandbluepilled** [comment]
fix removed
+1

**Comment by Manly-Dwarf** [comment]
eh still no
not convinced they'll really do anything

**Comment by MaterialisticThings** [comment]
+1

**Comment by zipzipzapityzoo** [comment]
+1 I like it but it certainly is more rp based, which is unlikely. I think they should have less gear than sec officers though, like batons. Police batons are dank, and a decent low escalation non lethal weapon. Having less gear and access though might make them both somewhat useless and purposeless as a job

**Comment by TrickyDickRichard** [comment]
miami vice officer

**Comment by Manly-Dwarf** [comment]
i still really like the idea but i fail to see it going anywhere in its current execution
my only ideas were not liked by the GMs (or rather lunaman, who has horrible taste any way)

**Comment by diegoxt45** [comment]
what does this job do EXACTLY

**Review by yoyobatty — DISMISSED** [review]

**Comment by Manly-Dwarf** [comment]
we still have no idea what we're doing with this

**Comment by wheezl** [comment]
merge this pr you hack fraud headmin

**Comment by TrickyDickRichard** [comment]
This PR has my sprites in it. Therefore, it is epic.

**Comment by robloxgothgf** [comment]
epic hall monitor +1

**Comment by ghost** [comment]
basically lawyer 2. for more uniqueness,  make the vice officer a monkey.  a roundstsrt monkey role,  a machine of law

**Comment by mkikt4743** [comment]
no one is gonna play this it's sec

**Comment by MrEpicSax** [comment]
I want this. Hurry up.

**Comment by TrickyDickRichard** [comment]
> I want this. Hurry up.

I had a hand in this PR and I just want to make this clear. The only people that get to type with PROPER GRAMMAR in this PR are the CONTRIBUTORS, so unless you got some CODE or SPRITES, NEVER speak in proper grammar in a PR of MINE, OR ANYONE ELSE'S PR. Do I make myself clear, you ABSOLUTE IDIOT?

**Comment by MrEpicSax** [comment]
> 
> 
> > I want this. Hurry up.
> 
> I had a hand in this PR and I just want to make this clear. The only people that get to type with PROPER GRAMMAR in this PR are the CONTRIBUTORS, so unless you got some CODE or SPRITES, NEVER speak in proper grammar in a PR of MINE, OR ANYONE ELSE'S PR. Do I make myself clear, you ABSOLUTE IDIOT?

AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA!!!! OOOOOOooooOOOOoooooOOOOOhhhhHHHHHH!!!! WHYYY WOULD YOU SAY THAT?!!!

**Comment by TrickyDickRichard** [comment]
> > > I want this. Hurry up.
> > 
> > 
> > I had a hand in this PR and I just want to make this clear. The only people that get to type with PROPER GRAMMAR in this PR are the CONTRIBUTORS, so unless you got some CODE or SPRITES, NEVER speak in proper grammar in a PR of MINE, OR ANYONE ELSE'S PR. Do I make myself clear, you ABSOLUTE IDIOT?
> 
> AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA!!!! OOOOOOooooOOOOoooooOOOOOhhhhHHHHHH!!!! WHYYY WOULD YOU SAY THAT?!!!

You fucking IMBECILE. You absolute fucking MONGOLOID. I asked you a YES OR NO QUESTION, and not only did you NOT ANSWER, you PROCEED TO TYPE IN CORRECT FUCKING GRAMMAR!!! GET OFF OF MY FUCKING PR RIGHT FUCKING NOW!!!

**Comment by MrEpicSax** [comment]
> 
> 
> > > > I want this. Hurry up.
> > > 
> > > 
> > > I had a hand in this PR and I just want to make this clear. The only people that get to type with PROPER GRAMMAR in this PR are the CONTRIBUTORS, so unless you got some CODE or SPRITES, NEVER speak in proper grammar in a PR of MINE, OR ANYONE ELSE'S PR. Do I make myself clear, you ABSOLUTE IDIOT?
> > 
> > 
> > AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA!!!! OOOOOOooooOOOOoooooOOOOOhhhhHHHHHH!!!! WHYYY WOULD YOU SAY THAT?!!!
> 
> You fucking IMBECILE. You absolute fucking MONGOLOID. I asked you a YES OR NO QUESTION, and not only did you NOT ANSWER, you PROCEED TO TYPE IN CORRECT FUCKING GRAMMAR!!! GET OFF OF MY FUCKING PR RIGHT FUCKING NOW!!!

:skull_and_crossbones:

**Review by AnthonyJNgo — APPROVED** [review]
Pretty simple changes, everything looks good.

**Comment by AnthonyJNgo** [comment]
This seems pretty popular with players and co, I'm a bit in agreement with KBB here in that I'm not sure how exactly this will play out. Overall though, this seems harmless enough, just a sec officer role with a bit of extra flair so I'm going to go ahead and merge it and worst case scenario we just do a revert.

**Pull request merged**